src/trackedit/api/projectapi.cpp

```
@@ -0,0 +1,126 @@
+        LOGE() << "trackIndex out of range: " << trackIndex << " (have " << ids.size() << " tracks)";
+        return {};
+    }
+    auto clipList = prj->clipList(ids[trackIndex]);
+    Clips clips(clipList.begin(), clipList.end());
+    std::sort(clips.begin(), clips.end(), [](const Clip& a, const Clip& b) {
+        return a.startTime < b.startTime;
+    });
+    return clips;
+}
+
+int ProjectApi::trackCount() const
```

Cppcheck (2.20.0)

[style] The function 'trackCount' is never used.

(unusedFunction)